src/main/g8/default.properties

```diff
@@ -1,3 +1,5 @@
 name=Cats Effect 3 Quick Start
 package=com.example
 scala3 = no
+testlib-use-cats-effect-testing-specs2=yes
```

Comment:
I would default all the testing libs to `no`, it'll be easier to enable just the one the user wants. Or we pick one as a recommendation... what do you think?

Reply:
I will work just fine if all the libs are enabled. But I think one testing libraries should be enabled by default. I don't have any preference towards any of the testing libraries

Reply:
Let's use MUnit and see if people complain.